lib/logatron/logatron.rb

```diff
@@ -50,12 +50,28 @@ def debug(msg)
       logger.debug(msg)
     end
 
+    # Deprecated; use request_info/parse_request_info for transportation-layer neutral methods.
     def http_headers
       {
-        'X-Ascent-Log-Id' => msg_id
+        'X-Ascent-Log-Id' => msg_id,
+        'X-Ascent-Site' => site
       }
     end
 
+    def request_info
+      {
+        message_id: msg_id,
+        site: site
+      }
+    end
+
+    def parse_request_info(info)
```

Comment:
This is not "parsing". A parse function takes a string of tokens and produces a structured representation of the information contained in the tokens. >90% of the time someone in our group says "parse" it is the wrong word. You'd be better off making this `request_info=` to be consistent with the `site/site=/msg_id/msg_id=` methods that it delegates to.